epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -123,8 +170,7 @@ public void bind(T view, List<Object> payloads) {
    *                             process, and follows the same general conditions for all
    *                             recyclerview change payloads.
    */
-  public void bind(T view, EpoxyModel<?> previouslyBoundModel) {
-    bind(view);
+  public void bindChanges(@NonNull T view, @NonNull EpoxyModel<?> previouslyBoundModel) {
```

Comment:
just to be clear, you're proposing a breaking change of renaming `bind` to `bindChanges`? if we do do that it would probably be good to do that one change it its own PR for clarity and since it will require updating some tests